docs/4.templates/6.template-nlu.mdx

```diff
@@ -30,8 +30,7 @@ npm install @botonic/plugin-nlu
 >Followed by: `npm install @botonic/plugin-nlu `
 
 ### 2. Add Utterances and Intents
-There are many ways that a user can express their intent. For example the Utterances "Hello", "Hi", and "Good morning" are all examples
-of a Greeting intent. 
+A user can express his intent in different ways. For example the Utterances "Hello", "Hi", and "Good morning" are all examples of a Greeting intent. 
```

Comment:
I don't like using male gender in docs. imho I prefer using feminine or plural

Reply:
I removed "his" to avoid the problem.